Assets/Mediapipe/Samples/Common/Scripts/Solution.cs

```diff
@@ -72,7 +72,33 @@ public virtual void Stop()
     protected static void SetupScreen(RawImage screen, ImageSource imageSource)
     {
       screen.rectTransform.sizeDelta = new Vector2(imageSource.textureWidth, imageSource.textureHeight);
-      screen.rectTransform.localEulerAngles = imageSource.rotation.Reverse().GetEulerAngles();
+      screen.rectTransform.localEulerAngles = imageSource.rotation.Reverse().GetEulerAngles();
+
+      //Check if the screen is flipped on mobile and flip it if it is the front camera

```

Comment:
Always insert a space character after `//`.
```suggestion
      // Check if the screen is flipped on mobile and flip it if it is the front camera
```

The same goes for other parts.